examples/tree_1d_dgsem/elixir_advection_doublefloat.jl

```diff
@@ -50,8 +52,10 @@ callbacks = CallbackSet(summary_callback,
 ###############################################################################
 # run the simulation
 
-sol = solve(ode, CarpenterKennedy2N54(),
-            dt = 42.0, # `dt` does not need to be in higher precision
+sol = solve(ode, DP8(),
```

Comment:
Why did you choose the methods of Feagin and DP8 in the examples? Would it make sense to use the same method in both?

Reply:
So I chose them to match the spatial order. Concerning the order, there is no real reason why I chose 8 besides 14 (which is the highest for which an explicit integrator exists in OrdinaryDiffEq.jl).

Reply:
👍